dependency-check-maven-config.xml

```diff
@@ -200,4 +229,258 @@
+        <cve>CVE-2014-0085</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to org.apache.geronimo:geronimo which is not a dependency of any of these artifacts.
+
+            geronimo-activation_1.1_spec-1.1.jar
+            geronimo-javamail_1.4_spec-1.7.1.jar
+            geronimo-jms_1.1_spec-1.1.1.jar
+            geronimo-jms_2.0_spec-1.0-alpha-2.jar
+            geronimo-servlet_3.0_spec-1.0.jar
+            geronimo-stax-api_1.0_spec-1.0.1.jar
+            apache-karaf-4.1.2.tar.gz: apache-karaf-4.1.2.tar: org.apache.servicemix.specs.saaj-api-1.3-2.9.0.jar/META-INF/maven/org.apache.geronimo.specs/geronimo-saaj_1.3_spec/pom.xml
+            apache-karaf-4.1.2.tar.gz: apache-karaf-4.1.2.tar: org.apache.servicemix.specs.jaxws-api-2.2-2.9.0.jar/META-INF/maven/org.apache.geronimo.specs/geronimo-jaxws_2.2_spec/pom.xml
+        </notes>
+        <cve>CVE-2011-5034</cve>
+        <cve>CVE-2008-0732</cve>
+        <cve>CVE-2006-0254</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to a frontend tool called jCore before 1.0pre2.
+            It does not apply to the artifact that OWASP is reporting.
+            apache-mime4j-core-0.7.2.jar
+        </notes>
+        <cve>CVE-2012-4231</cve>
+        <cve>CVE-2012-4232</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            This CVE applies to Apache-SSL 1.3.28+1.52 and earlier. It does not apply to the artifact that OWASP is reporting.
+            apache-karaf-4.1.2.tar.gz: apache-karaf-4.1.2.tar: org.apache.servicemix.bundles.not-yet-commons-ssl-0.3.11_1.jar
+        </notes>
+        <cve>CVE-2004-0009</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs apply to "Apache QPID 0.30" and previous versions. This particular maven artifact is a new qpid artifact.
+            Its first release came after all of these CVEs.
+            qpid-jms-client-0.11.1.jar
+        </notes>
+        <cve>CVE-2015-0223</cve>
+        <cve>CVE-2015-0224</cve>
+        <cve>CVE-2013-1909</cve>
+        <cve>CVE-2012-4460</cve>
+        <cve>CVE-2012-4459</cve>
+        <cve>CVE-2012-4458</cve>
+        <cve>CVE-2012-4446</cve>
+        <cve>CVE-2012-3467</cve>
+        <cve>CVE-2012-2145</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            Applies to an LDAP class that we do not use in the system. Also, this jar is only used in tests.
+            groovy-all-2.4.7.jar
+        </notes>
+        <cve>CVE-2016-6497</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            The CVE applies to xzgrep, not the xz artifact.
+            tika-bundle-1.14.0_2.jar: xz-1.5.jar
+            xz-1.6.jar
+        </notes>
+        <cve>CVE-2015-4035</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            These CVEs do not apply to this version of bzip2
+            bzip2-0.9.1.jar
+        </notes>
+        <cve>CVE-2011-4089</cve>
```

Comment:
❓ Are we saying it doesn't apply to version `bzip2-0.9.1.jar`? If so, it looks like this one applies to versions all the way up to 1.0.4 (https://nvd.nist.gov/vuln/detail/CVE-2011-4089) and the next applies to versions up to 1.0.5 (https://nvd.nist.gov/vuln/detail/CVE-2010-0405). Doesn't look like there's a fix for the last one though...

Reply:
Versions 1.0.4 and 1.0.5 apply to this bzip2 (last release was in 2010): http://www.bzip.org/
The bzip being used in DDF is https://mvnrepository.com/artifact/org.itadaki/bzip2/0.9.1, a different artifact that was released in 2015.